### ISSUE TITLE
Rotation Listener

### ISSUE THREAD

**Issue by sbrokalis** [react-pdf-viewer/react-pdf-viewer#959]
Is there any way to listen for rotation changes in the default layout?

**Comment by phuoc-ng** [comment]
@sbrokalis 
Update: The listener is added to the latest code, and will be available in the next version (v3.2.0).

```js
<Viewer
    onRotate={({ doc, rotation }) => {
        // `doc` is the current document
        // `rotation` is the latest rotation value
    }}
/>
```

**Comment by phuoc-ng** [comment]
@sbrokalis 
Please upgrade to the v3.2.0: https://react-pdf-viewer.dev/updates/
